src/types/globalTypes.ts

```diff
@@ -649,6 +649,13 @@ export enum OrderStatus {
   UNFULFILLED = "UNFULFILLED",
 }
 
+export enum OrderStatusType {
+  error = "error",
+  neutral = "neutral",
+  alert = "alert",
+  success = "success"
```

Comment:
Lower case enum was generated by codegen or added by hand?

Reply:
By hand, change to uppercase?

Reply:
@mmarkusik file is build based on graphql schema file. If you want to add new enums, add it there and then run `npm run build-types`.